### PR TITLE
new version of npm lockfile

### PR DESCRIPTION
**Pull request by stephenwaite**
Fixes #
been noticing this altered file in the easy dev docker, it's altered after the build process completes
finally decided to commit it and see what others think is going on

#### Short description of what this resolves:
believe that when @sjpadgett created this he was on a different version of npm than what is installed in the easy dev docker
here's a related [so thread](https://stackoverflow.com/questions/64813775/is-there-any-way-to-fix-package-lock-json-lockfileversion-so-npm-uses-a-specific)

#### Changes proposed in this pull request:
update ccda `package-lock.json`

### CODE REVIEW

**Comment by adunsulag** [comment]
So the lock file does come from a newer version of npm (node > 12, I believe) as it changed how the lock files are built.

**Comment by bradymiller** [comment]
I'd prob drop that package-lock.json completely and then start up a new easy dev docker env (dev docker uses current stable nodejs version of 16) and then commit the newly created lock file (as long as nothing breaks on the build). Then should get rid of the issue of showing altered lock file after start up the docker.

**Comment by sjpadgett** [comment]
> dev docker uses current stable nodejs version of 16

This version is broken on windows. does ccda service install okay on linux? Last tested version I know works was LTS 14.xx

**Comment by stephenwaite** [comment]
appears to be fine
Screenshot from 2022-01-14 10-18-01

**Comment by sjpadgett** [comment]
You have to generate a CCDA to test by doing search to grab available patients then click doc button..
OpenEMR-v6-1-0

 To yield

image

**Comment by stephenwaite** [comment]
works on https://ten.openemr.io/a/openemr/ running `Alpine 3.13 - uses php 7.4` and think have found probs with php8

**Comment by stephenwaite** [comment]
will test again but think this is working now for easy-dev

one thing noticed is creation of ccda files for the test pt outside of the documents directory 
Screenshot from 2022-01-14 19-44-14

**Comment by stephenwaite** [comment]
going to replace the `-xsd` package with  "libxmljs2": "^0.29.0

File: ccdaservice/package.json [hunk 1]
```diff
@@ -8,9 +8,10 @@
     "license": "GPL-3.0",
     "dependencies": {
         "file": "./oe-blue-button-generate",
+        "libxmljs2": "^0.29.0",
         "lodash": "4.17.21",
         "net": "1.0.2",
+        "uuid": "^8.3.2",
```
Comment (bradymiller) [inline ccdaservice/package.json:14]: for both above packages, recommend pinning the specific version

**Comment by bradymiller** [comment]
those ccda files outside documents are problematic since they will be able to be viewed by users that are not authenticated (ie. they can go directly to that path via a web browser and view the files in that directory without logging in)

**Comment by stephenwaite** [comment]
oscar

**Comment by sjpadgett** [comment]
At some point we have to lock in nodejs versions. For linux, the libxmljs library tends to offer and keep up with the compiled versions however on windows after nodejs LTS 14, then library has to be compiled with windows build tools. A lengthy and unstable process depending on environment.

I thought this was clear when I brought in the locked packaged and was documented. Let's not forget MacOS as well.
We can keep this and nice work @stephenwaite but I'll have to come up with solution for windows.

**Review by bradymiller — APPROVED** [review]

**Comment by bradymiller** [comment]
2DV

**Pull request merged**